src/Tester/Tester.csproj

```diff
@@ -110,6 +115,10 @@
       <Link>OrleansConfigurationForTesting.xml</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\OrleansTestingHost\OrleansDependencyResolverConfigurationForTesting.xml">
```

Comment:
I put that xml to the project where all other xml files are. That is the reason. But of course I can move it!